solr/contrib/ltr/src/test/org/apache/solr/ltr/TestRerankBase.java

```diff
@@ -312,7 +312,7 @@ public static void loadModels(String fileName) throws Exception {
   }
 
   public static LTRScoringModel createModelFromFiles(String modelFileName,
```

Comment:
overload that defaults featureFileName to _DEFAULT_ so callers dont have to manually put in null every time?